desktop/cmp/input/DateInput.js

```diff
@@ -308,26 +328,35 @@ class Model extends HoistInputModel {
         return date;
     }
 
-    getFormat() {
-        const {timePrecision} = this,
-            {formatString} = this.props;
+    getFormatString() {
+        const {formatString} = this.props;
+
+        return formatString ?? this.addTimeFmt('YYYY-MM-DD');
+    }
+
+    getParseStrings() {
+        const {formatString, parseStrings} = this.props;
+
+        if (parseStrings) return parseStrings;
+
+        const ret = ['YYYY-MM-DD', 'YYYYMMDD', 'YYYY/MM/DD', 'YYYY-M-DD', 'YYYY/M/DD'].map(s => this.addTimeFmt(s));
```

Comment:
Would anyone use the slash-delimited variations, with the year coming first? "2020/09/17" - is that a thing?

Any sense we should also support `L` - local date format - for e.g. `9/17/2020` in the US? (Unclear to me if this can be used along with the timeFmts - not anything we need to do now, but mentioning it in case that was more what you were looking to accept)